typescript/web/src/components/pagination-toolbar/pagination-toolbar.tsx

```diff
@@ -0,0 +1,100 @@
+import {
+  useColorModeValue as mode,
+  Text,
+  Grid,
+  HStack,
+  Center,
+} from "@chakra-ui/react";
+import { ToggleButtonGroup, ToggleButton } from "../toggle-button-group";
+import { PaginationTool } from "../pagination-tool";
+
+interface PaginationToolbarProps {
+  itemsTotalCount: number;
+  itemsPerPageOptions: number[];
+  currentPage: number;
+  itemsPerPage: number;
+  onChange: (page: number, itemsPerPage: number) => void;
+}
+
+export const PaginationToolbar = (props: PaginationToolbarProps) => {
+  const {
+    itemsTotalCount,
+    itemsPerPageOptions,
+    currentPage,
+    itemsPerPage,
+    onChange: setPagesAndItemsPerPages,
+  } = props;
+
+  const itemsPerPageStrToNum = itemsPerPageOptions.reduce<
+    Record<string, number>
+  >((record, num) => {
+    return {
+      ...record,
+      [num.toString()]: num,
+    };
+  }, {} as Record<string, number>);
+
+  const skip = (currentPage - 1) * itemsPerPage;
+  const maxPages = Math.ceil(itemsTotalCount / itemsPerPage);
+
+  if (maxPages && currentPage > maxPages) {
+    setPagesAndItemsPerPages(maxPages, itemsPerPage);
+  }
+
+  return (
+    <Grid
+      pos="fixed"
+      bottom="0px"
+      w="100%"
+      h="44px"
+      zIndex={2}
+      bg={mode("white", "gray.800")}
+      paddingLeft="32px"
+      paddingRight="32px"
+      templateColumns="repeat(3, 1fr)"
+    >
+      <HStack alignItems="center">
+        <Text>Items:</Text>
+        <ToggleButtonGroup
+          value={itemsPerPage.toString()}
+          onChange={(value) => {
+            setPagesAndItemsPerPages(1, itemsPerPageStrToNum[value]);
+          }}
+          isAttached
+          variant="outline"
+        >
+          {Object.keys(itemsPerPageStrToNum).map((itemsPerPageOption) => (
+            <ToggleButton
+              size="sm"
+              key={`imagesPerPageButton-${itemsPerPageOption}`}
+              value={itemsPerPageOption}
+              description={`${itemsPerPageOption} images per page`}
+              pointerEvents="initial"
+              borderColor={mode("gray.100", "whiteAlpha.300")}
+              _checked={{
```

Comment:
This is the same as the default values set in the ToggleButton component no? I would say you don't need to pass it